bundle-size/api.js

```diff
@@ -181,6 +218,54 @@ function extraBundleSizesSummary(otherBundleSizeDeltas, missingBundleSizes) {
   );
 }
 
+/**
+ * Choose the set of all the potential approver teams into a single team.
+ *
+ * Could end up choosing the fallback set defined in the `.env` file, even if
+ * the fallback set is not in the input.
+ *
+ * @param {Array<Array<string>>} allPotentialApproverTeams all the potential
+ *   teams that can approve this pull request.
+ * @param {Logger} log logging function/object.
+ * @param {number} pullRequestId the pull request ID.
+ * @return {Array<string>} the selected subset of all potential approver teams
+ *   that can approve this bundle-size change.
+ */
+function choosePotentialApproverTeams(
+  allPotentialApproverTeams,
+  log,
+  pullRequestId
+) {
+  let potentialApproverTeams;
+  switch (allPotentialApproverTeams.length) {
+    case 0:
+      potentialApproverTeams = [];
```

Comment:
This would not catch cases where `foo.js` requires approval from `wg-infra` or `wg-runtime` and `src/bar.js` requires approval from `wg-runtime`.

Reply:
Correct, this was a deliberate choice to make the codebase here simpler at the cost of super-accurate team matching. Depending on how this ends up being used by the different teams, I could be convinced to fix that - but I have a feeling most files will be assigned to individual groups and there won't be a lot of overlap